slices/fontconfig-config.yaml

```diff
@@ -7,6 +7,9 @@ slices:
   config:
     essential:
       - fonts-dejavu-core_config
+      - fonts-noto-core_config
+      - fonts-texgyre_config
+      - fonts-urw-base35_config
```

Comment:
The `fontconfig-config` package only requires one fonts package, is there any reason to be adding those three additional ones?
![image](https://github.com/user-attachments/assets/02eb9688-e3ce-4b40-936a-64d6d7792960)

Reply:
@HadrienPatte primarily due to the fact I haven't looked careful enough on the `apt show` output. Can leave only fonts-dejavu-core as a primary dependency _and_ keep the rest of slices just for the sake of having them in the releases.